tables/endpointsecurity/src/endpointsecurityservice.cpp

```diff
@@ -51,10 +58,18 @@ Status EndpointSecurityService::exec(std::atomic_bool &terminate) {
       continue;
     }
 
+    if (!d->file_events_table) {
+      std::this_thread::sleep_for(std::chrono::seconds(1U));
+      continue;
+    }
```

Comment:
Can you combine this with the block for `process_events_table` above? It would also be great if we would emit something to the log here so users can detect us waiting long times (e.g., `Table XYZ not created yet, sleeping for N seconds`).